dali/pipeline/operator/op_schema.h

```diff
@@ -454,6 +460,14 @@ class DLL_PUBLIC OpSchema {
     return *this;
   }
 
+  DLL_PUBLIC inline OpSchema &AddDeprecatedArg(const std::string& arg_name,
+                                               const std::string& deprecation_str,
+                                               const std::string& renamed_to,
+                                               bool ignore = true) {
+    deprecated_arguments_[arg_name] = {deprecation_str, renamed_to, ignore};
+    return *this;
+  }
```

Comment:
I think you can add overloads:
```
AddDeprecatedArg(arg_name, bool ignore=false, optional_message="")
AddDeprecatedArg(arg_name, replaced_by, bool ignore=true, optional_message="");
```
I would add a doc that one is for removing arguments and the other for replacing.

Reply:
Also maybe `DeprecateArg` instead?